src/styles/blocks/heder.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+
+  &__picture {
+    width: 40px;
+    height: 40px;
+    padding: 0;
+  }
+
+  &__navlist-linkstyle {
+    display: inline-block;
+    color: $main-text-color;
+    height: 60px;
+    line-height: 60px;
+    font-size: 12px;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-weight: 500;
+    position: relative;
+    padding: 0;
+    transition: color 0.3s ease;
+  }
+
+  &__navlist {
+    list-style: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  &__navlist-items {
+    display: inline-block;
+    margin-right: 20px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+}
+
+.is-active {
```

Comment:
The class name `.is-active` is a modifier and should be written as `&--is-active`.